packages/jest-types/src/Global.ts

```diff
@@ -106,10 +106,15 @@ export interface HookBase {
   (fn: HookFn, timeout?: number): void;
 }
 
+export interface Failing<T> {
+  (testName: TestNameLike, fn: T, timeout?: number): void;
+  each?: Each<TestFn>;
```

Comment:
`each` shouldn't be optional, should it?

Reply:
Removed optional in this [commit](https://github.com/facebook/jest/pull/13142/commits/74e5a41536b5e3691498dc5d57e54181e5c0a2fd)